doc/quick-start.adoc

```diff
@@ -6,7 +6,8 @@ Download the latest release of Kadalu Kubectl plugin using,
 
 [source,console]
 ----
-curl -LO https://github.com/kadalu/kadalu/releases/download/0.8.2/kubectl-kadalu
+
```

Comment:
This is very important. Thanks for this.

While at it (or later), can we merge below 20+ lines and make a single block which can be copy pasted in one shot? This helps to make steps easier for people going through docs, mainly because the steps are self-explanatory, and the extra sentences are not adding any value.